src/v/net/tests/connection_rate_test.cc

```diff
@@ -12,13 +12,144 @@
 #include "seastarx.h"
 
 #include <seastar/core/lowres_clock.hh>
+#include <seastar/core/manual_clock.hh>
 #include <seastar/core/sleep.hh>
 #include <seastar/core/when_all.hh>
 #include <seastar/net/inet_address.hh>
 #include <seastar/testing/thread_test_case.hh>
 
 #include <boost/test/tools/old/interface.hpp>
 
+#include <algorithm>
 #include <chrono>
 #include <unordered_map>
 
+SEASTAR_THREAD_TEST_CASE(general_rate_test) {
```

Comment:
@VadimPlh please document your changes more thoroughly. The previous commit says "net: Delete unstable test for connection_rate" and in this commit the only message is "net: Added test for conection_rate limit".

There is no indication in the commit regarding why the previous version was unstable and why the new one is better / or how it fixes things.